tools/cli/commands/generate.js

```diff
@@ -266,6 +266,7 @@ export async function generateProject(
 
 	switch ( answers.type ) {
 		case 'package':
+			await searchReplaceInFolder( projDir, 'PACKAGE-NAME', normalizeSlug( answers.name ) );
```

Comment:
CI doesn't seem to like the uppercase "PACKAGE-NAME", and I don't see any instances of "package-name". Should be safe to just go with that instead.

I wonder if we should put this call up near line 252 instead though.

Reply:
There's a lower case `package-name` in `.phpcs.dir.xml`.

We can change example.php to lowercase as well

Reply:
> I wonder if we should put this call up near line 252 instead though.

You mean doing it for all project types?

Reply:
and there are some upper-case in example.php

Reply:
> There's a lower case package-name in .phpcs.dir.xml.

You mean the ones you're trying to make uppercase in this PR, that are making phpcs unhappy?

> You mean doing it for all project types?

Yes. Why not?

> and there are some upper-case in example.php

We're not replacing those currently, although we probably should. They could be lowercased too, or we could replace both upper- and lower-case versions of the string.